x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/rest/RestDeleteRollupJobAction.java

```diff
@@ -47,7 +55,7 @@ protected RestStatus getStatus(DeleteRollupJobAction.Response response) {
 
     @Override
     public String getName() {
-        return "rollup_delete_job_action";
+        return "delete_rollup_job";
```

Comment:
I've noticed a few of these changes that came along with my unrevert. Are these changes compatible with a mixed version cluster?

Reply:
That's a good question... I'm not sure.  I don't really know how REST request names are used (or if they are just for display/task purposes?)